### ISSUE TITLE
[Regression] Skylanders imaginators, Skylanders superchargers (#14414)

### ISSUE THREAD

**Issue by DarkMessiahx** [RPCS3/rpcs3#15088]
### Quick summary

These 2 games play the logo/intro videos and then hang on an infinite black screen.

### Details

These games (Skylanders superchargers, Skylanders imaginators) play the logo/intro videos and then hang on an infinite black screen. These 2 games worked on version 0.0.29-15422, but had the issue as described on 0.0.29-15427

Note: on the build history page there are 4 builds missing between these 2 builds and 15426 is crossed out, i was told on the discord that those do not count.

### Build with regression

v0.0.29-15427

### Attach two log files

working
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/14031808/RPCS3.log)

not working
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/14031837/RPCS3.log)

### System configuration

CPU: Intel I7 9700
GPU: Nvidia RTX 4070
RAM: 48gb DDR4
RPCS3 runs from SSD

**Comment by DarkMessiahx** [comment]
yes, default settings, i removed the custom profiles, spyro's adventure, giants, swap force and trap team runs fine on the latest version, superchargers and imaginators gets stuck on black screen after logos and intro cinematics. both games run fine on v0.0.29-15422, but not on the latest version or on any versions after v0.0.29-15427

**Comment by digant73** [comment]
apparently regressed by #14414.

@DarkMessiahx is that still an issue?

**Comment by digant73** [comment]
please an update on current build
